Ping arduino

cb218 opened the pull request:
Created arduino/monitor.go which pings arduino every hour.
If a ping is successful, the lastArduinoReachableTimestamp is updated.

rFronteddu reviewed (changes requested):
I am not sure I understand why you are using GRPC. GRPC should be used to invoke a task in another device. The code as I wrote it was designed to receive a request from a server and perform a ping.

What you want to do here is to periodically ping the sensor board. Why did you feel like you needed grpc?

Also, I think we should avoid architecture names as packages or file names. In go, using architecture names can signify that you are writing architecture-specific code. 

I think what you wanted to do here is use the icmp prober and not the pinger task.

cb218 commented:
I converted the grpc ping to icmp in the latest commit.
I also removed all architecture names, except for in the report I send "LastArduinoReachableTimestamp".
"LastArduinoReachableTimestamp" is the integer value in seconds of Time.Now() - the last reachable timestamp.
